### DISCUSSION TITLE
Navigation: "10 secs rewind" and "10 secs forward"

### DISCUSSION THREAD

**Discussion by jman-schief** [FossifyOrg/General-Discussion#572] [Ideas]
Hi!

Sometimes it's a bit hard to find the exact spot where I want to start listening a podcast and find myself wishing there was a control to skip a little bit forward or rewind. Could also be useful to jump around.

What do you folks think about it?

Thanks

**Comment by Aga-C** [comment]
See: https://github.com/FossifyOrg/Music-Player/issues/86#issuecomment-2297318467.

**Reply by jman-schief** [reply]
I must be blind :laughing: 

thanks!
